docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -181,7 +181,7 @@ window as the previous step.
 
 .. code-block:: console
 
-   user@validator$ sudo sawadm keygen
+   $ sudo sawadm keygen
```

Comment:
The `user@validator` prompt is a useful indicator that the commands should be run for validator only. We're following the same standard below for the client term window. Instead of removing it, the rest of the document should adopt this standard. Especailly the genesis and this sawadm commands are very specific for validators only.

Reply:
+1 for consistency

Reply:
I added use@validator to all the preceding prompts, as requested.